### ISSUE TITLE
[Windows?] Building blade backend with opengl fails

### ISSUE THREAD

**Issue by MolotovCherry** [zed-industries/zed#16677]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Using `RUSTFLAGS="--cfg gles"` on a windows build fails compilation due to a dependency error. Untested on other OSs, but definitely fails for me when building for windows.
```rs
Error:     --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\khronos-egl-5.0.0\src\lib.rs:2016:16
     |
2016 |                       let ptr = (&symbol.into_raw().into_raw()) as *const *mut _ as *const unsafe extern "C" fn($($atype ),*) -> $rtype;
     |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
2210 | / api! {
2211 | |     EGL1_0 : "1_0" {
2212 | |         fn eglChooseConfig(
2213 | |             display: EGLDisplay,
...    |
2362 | |     }
2363 | | }
     | |_- in this macro invocation
     |
     = note: this error originates in the macro `api` (in Nightly builds, run with -Z macro-backtrace for more info)
For more information about this error, try `rustc --explain E0606`.
error: could not compile `khronos-egl` (lib) due to 5 previous errors
Error: warning: build failed, waiting for other jobs to finish...
Error: The process 'C:\Users\runneradmin\.cargo\bin\cargo.exe' failed with exit code 101
```

### Environment

n/a

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by MatinAniss** [comment]
The actual issue has been fixed in blade https://github.com/kvark/blade/pull/156, but Zed is still using a older commit.

**Comment by MolotovCherry** [comment]
> The actual issue has been fixed in blade [kvark/blade#156](https://github.com/kvark/blade/pull/156), but Zed is still using a older commit.

Thanks for the hint. I'll make a PR for this.
